### ISSUE TITLE
Feature request: vox can activate virtual environments under arbitrary paths

### ISSUE THREAD

**Issue by halloleo** [xonsh/xonsh#2935]
At the moment `vox` can only activate virtual environments under $VIRTUALENV_HOME. 

I guess the model behind this is, that `vox` not only *activates* venvs, but *manages* the whole life cycle of venvs from creation to to usage to deletion.

But why this limitation? If the `activate` command would take optionally a path, it could activate an virtual environment under this directory as well, couldn't it? Then we could activate "foreign" venvs created before and independently of xonsh.

If this sounds feasible to core members, I'm happy to look into the code for this.

**Comment by scopatz** [comment]
I think we'd be happy to have this as a feature! PRs very welcome here.

**Comment by eode** [comment]
..userdocs for vox then?

**Comment by halloleo** [comment]
Tested this, and indeed it works: In the CLI you can give `vox activate` the path to a venv and it will activate it.

I'll try to add this to the help strings of the vox command.

**Comment by halloleo** [comment]
Mmmmh, I tried to correct the problem with my pull request #2940, I closed it and created a new PR #2942, but this still doesn't satisfy all CI checks:

![screen shot 2018-11-17 at 11 18 58](https://user-images.githubusercontent.com/1521852/48653655-dcf80c80-ea5a-11e8-930b-839e1ac751df.png)

Any idea how rectify this? I think the RST syntax in the news file is correct now. I'm not too familiar with GitHub's CI system.
